controller/src/main/java/org/jboss/as/controller/ParallelBootOperationContext.java

```diff
@@ -324,6 +324,11 @@ void awaitServiceContainerStability() throws InterruptedException {
     }
 
     @Override
+    void publishCapabilityRegistry() {
+        primaryContext.publishCapabilityRegistry();
```

Comment:
This should be a no-op. The primary context controls publication when everything is ready; the ParallelBootOperationContexts only know what has happened with their bit.